### PR TITLE
New blog post on Micro Services

### PR DESCRIPTION
**Pull request by crystoll**
Attempt for my first blog post here, let's see if I got it right. This post is about Solita new employees exercise and Micro services, and I appreciate if someone checks I didn't mess anything up.

### CODE REVIEW

File: _posts/2015-02-16-microservices.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Every new employee in the development will need/get to do an exercise in coding, no matter what they will end up doing here. Idea is to pass on knowledge and ideas both ways, on tools, technologies and architecture, and I think it's a great idea. So I wanted to write a little blog post on my experiences.
+
+I am pretty experienced in all things Java, and recently also in Angular, so I wanted a mix of something familiar - but also something new, to raise the bar a bit and learn something new. For the familiar base I chose Spring Boot and AngularJS - I'm pretty experienced with Spring after training it for major companies in Finland, and also using in various production systems along the way. I chose to base everything on Spring Boot because that part still has something new for me to learn. Maven was the solid basis for my builds, while Grunt handled whatever automation JavaScript side required.
+
+Spring Boot will get you a nice EE environment and servies in embedded container with just a little bit of Maven/Gradle build script. It can be deployed as a .jar file, and only requirement then is to have Java installed.
```
Comment (aautio) [inline _posts/2015-02-16-microservices.md]: servies -> services

File: _posts/2015-02-16-microservices.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+![Microservices container is up and running in 5 seconds](/img/microservices/springbootrun.png)
+
+I chose to mix things up by not using Bootstrap as my UI library, but instead to go for more mobile-like and puristic approach with Google Material Design philosophy and libraries. I also chose to use Java EE standards and generic APIs as much as possible, instead of coding against specific third party implementations.
+
+To make things more difficult/fun I chose a Macbook Pro as my laptop - being a veteran Windows/Linux user, this is a new experience and actually much fun. On the other hand I find myself often in the ever so familiar terminal window when things get hairy.
+
+I also made transition from Netbeans to IntelliJ - now I love netbeans, but IntelliJ is more common here, and I've heard a lot of great things about it, too. So Idea it is.
```
Comment (aautio) [inline _posts/2015-02-16-microservices.md]: Netbeans -> NetBeans, netbeans -> NetBeans

File: _posts/2015-02-16-microservices.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Microservices FTW
+author: arto
+excerpt: How to boot up and get some feedback fast
+---
+
+My name is Arto Santala. I've worked in many roles, including software developer, engineer, project manager, scrum master, and architect, as well as a trainer/coach - in companies like Tieturi, ICL, Finnair, and many others. While I've been involved in producing software since the mid-90's, and even before that, I came as new employee in Solita early this year, so it's an exciting new step for me.
```
Comment (pvto) [inline _posts/2015-02-16-microservices.md]: I came in as a new employee to Solita

come ... in - lienee ok merkityksessä "otettiin Solitalle"
tai sitten pelkkä come ... to

File: _posts/2015-02-16-microservices.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+Whenever there's a new employee in Software Development department, they will need/get to do an exercise in coding, no matter what they will end up doing here. Idea is to pass on knowledge and ideas both ways, on tools, technologies and architecture, and I think it's a great idea. Equally important is to get to know people, habits, and ways of working together - these also go both ways, since everybody bring along unique set of skills and experiences. So I wanted to write a little blog post on my experiences.
```
Comment (pvto) [inline _posts/2015-02-16-microservices.md]: Eka lause on vähän väljä, voisko tiivistää?  esim.  A new SW developer at Solita first gets to do an ...

The idea is to pass

everybody brings along a unique

File: _posts/2015-02-16-microservices.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+I am pretty experienced in all things Java, and recently also in Angular, so I wanted a mix of something familiar - but also something new, to raise the bar a bit and learn something new. For the familiar base I chose Spring Boot and AngularJS - I'm pretty experienced with Spring after training it for major companies in Finland, and also using in various production systems along the way. I chose to base everything on Spring Boot because that part still has something new for me to learn. Maven was the solid basis for my builds, while Grunt handled whatever automation JavaScript side required.
+
+Spring Boot will get you a nice EE environment and services in embedded container with just a little bit of Maven/Gradle build script. It can be deployed as a .jar file, and only requirement then is to have Java installed.
```
Comment (pvto) [inline _posts/2015-02-16-microservices.md]: an embedded container

the only requirement then

File: _posts/2015-02-16-microservices.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+![Material design is bold and simplistic, and scales well to and from mobile devices](/img/microservices/employeesapp.png)
+
+Actually, the microcontainer approach worked so well that a devil went into me and I decided to see if I can run the app inside Raspberry Pi v2 that I recently purchased. And it was lovely! ;) Just install Java into Pi, and run the packaged .jar - and Spring Boot takes care of everything. Gotta love the simplicity and elegance. It's just somewhat  different experience to deploying a J2EE .ear into Websphere Application Server....
```
Comment (pvto) [inline _posts/2015-02-16-microservices.md]: somewhat different an experience

File: _posts/2015-02-16-microservices.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Microservices FTW
+author: arto
+excerpt: How to boot up and get some feedback fast
+---
+
+My name is Arto Santala. I've worked in many roles, including software developer, engineer, project manager, scrum master, and architect, as well as a trainer/coach. I've worked for companies such as Tieturi, ICL, Finnair, and many others. While I've been involved in producing software since the mid-90's, and even before that, I came as new employee to Solita early this year, so it's an exciting new step for me.
```
Comment (Darep) [inline _posts/2015-02-16-microservices.md]: Voisko olla `I joined Solita as a new employee...`? Toi come tuntuu ainakin IMHO jotenki väärältä :)

Comment (bafeus) [inline _posts/2015-02-16-microservices.md]: screen shot 2015-02-19 at 15 57 39

**Pull request merged**